src/pages/HomePage/ListOfHouses/ListOfHouses.module.scss

```diff
@@ -50,6 +51,7 @@
 
   img {
     width: 100%;
+    max-width: 370px;
```

Comment:
Height could also be specified as it is possible to get this kind of photos:
![Screenshot from 2022-02-09 19-03-03](https://user-images.githubusercontent.com/47178494/153262213-9377fd7d-e290-467e-8fa3-45c92a4ed874.png)
We should definitely remove my example houses before the presentation but there could be some fallback for horizontal photos. To keep the proportions of a photo, you could use `object-fit: cover`. Changing `max-width` to just `width` will probably also be necessary.